ci-operator/step-registry/aws/upi-templates-generate/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- patrickdillon
+- wking
```

Comment:
Can you drop me?  I don't think I have much to contribute to UPI templates anymore, certainly not at a "likely to be able to provide timely review/direction" level.

Reply:
OK, will do.